src/main/runner.ts

```diff
@@ -190,7 +190,14 @@ export default class RunnerMain extends Runner {
               || this.findForgottenTestById(data.id);
             assert(test, `Couldn't find test by id: ${data.id}`);
 
-            this.emit(event, test, data.err);
+            const testErrors = (test as any).err;
```

Comment:
Please avoid using `as any` - this doesn't let us use TS advantages. As far as I understand `test` is [missing](https://github.com/mocha-parallel/mocha-parallel-tests/blob/4ba90b5d452e1362a8a8f6f123d004e7c779fb67/src/interfaces.ts#L24) `err` property? Please add it to `mocha` types and add a link here to your PR so we could track it.

Reply:
@1999 

Just to clarify:

Question 1:
You mean to add err property here?
https://github.com/types/mocha/blob/master/lib/global.d.ts#L172

It also complains about IHook, but I can't find it in mocha types.
![image](https://user-images.githubusercontent.com/3316019/52940926-b78bdf00-3378-11e9-8ff2-6632606f40bf.png)

Please, suggest what to do in this case.

Question 2:
Why can't we just add this property to interfaces within mocha-parallel-tests?

https://github.com/mocha-parallel/mocha-parallel-tests/blob/4ba90b5d452e1362a8a8f6f123d004e7c779fb67/src/interfaces.ts#L17

and

https://github.com/mocha-parallel/mocha-parallel-tests/blob/4ba90b5d452e1362a8a8f6f123d004e7c779fb67/src/interfaces.ts#L24

Reply:
Also how soon guys in mocha types will merge that change?